generators/client/templates/angular/src/main/webapp/_index.html

```diff
@@ -42,6 +44,7 @@
             document.getElementsByTagName('head')[0].appendChild(script)
         }
     </script>
+    <!-- <script src="sw-loader.js" />-->
```

Comment:
it need not be separate file, I think it can just be script inside a <script> tag like the google analytics code below. Sorry for not being clear enough earlier